libs/blockchain/migrations/3_metatransactionwallet.js

```diff
@@ -1,5 +1,7 @@
+const { NULL_ADDRESS } = require("@celo/base/lib/address");
+
 const MetaTransactionWallet = artifacts.require("MetaTransactionWallet");
 
 module.exports = async function (deployer, network, addresses) {
-  await deployer.deploy(MetaTransactionWallet);
+  await deployer.deploy(MetaTransactionWallet, true);
```

Comment:
Is this MTW for the relayers or for the user? If the latter, I think it should be `false`, no? Should we even be changing the version of the MTW we're using as compared with what's currently in Komenci? My feeling would be no.

Reply:
Yes this should be `false` I miss-read the comment in the contract. And yeah I also agree that we shouldn't redeploy this on mainnet but when I ran the migrations locally I added the bool to make it work.